docs/ci-cd/index.md

```diff
@@ -6,6 +6,12 @@
 tools/*
 ```
 
+## Introduction
+
+-   [Build Guide](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md)
```

Comment:
If you provide a URL, I wonder if a link like https://project-chip.github.io/connectedhomeip-doc/guides/BUILDING.html would result in a more easily readable item